package.json

```diff
@@ -18,7 +18,7 @@
     "Richard Chamberlain <richard_chamberlain@uk.ibm.com> (https://github.com/rnchamberlain)"
   ],
   "scripts": {
-    "test": "tap test/test*.js"
+    "test": "make lint && tap test/test*.js"
```

Comment:
Agreed, I don't see a reason to add a `Makefile`, it's another entry point to the module, which seems unnecessary as we already have a `package.json`.